.circleci/src/commands/restore_environment_variables.yml

```diff
@@ -0,0 +1,8 @@
+description: Sets the environment variables specified in the file 'environment_variables'. Make sure the file is persisted and has been attached.
+steps:
+  - run:
+      name: List environment variables
+      command: cat environment_variables
```

Comment:
Just to make sure, this command is especially used for debugging and stuff, right?